File: JavaScript/ordenamiento_seleccion.js
function selection(arr) {
    for (let i = 0; i < arr.length; i++) {
        let small = i;
        for (let j = i + 1; j < arr.length; j++) {
            if (arr[small] > arr[j]) {
                small = j;
            }
        }
        [arr[i], arr[small]] = [arr[small], arr[i]];
    }
}

function printArr(arr) {
    console.log(arr.join(" "));
}

// Generar arreglo aleatorio
let a = Array.from({ length: 5 }, () => Math.floor(Math.random() * 100) + 1);

console.log("Arreglo antes de ser ordenado:");
printArr(a);

selection(a);

console.log("\nArreglo después de ser ordenado:");
printArr(a);

File: Python/ordenamiento_seleccion.py
import random

def selection(a):
    for i in range(len(a)):
        small = i
        for j in range(i + 1, len(a)):
            if a[small] > a[j]:
                small = j
        a[i], a[small] = a[small], a[i]

def printArr(a):
    for i in range(len(a)):
        print(a[i], end=" ")
    print()

# Generar arreglo aleatorio
a = [random.randint(1, 100) for _ in range(5)]

print("Arreglo antes de ser ordenado:")
printArr(a)

selection(a)

print("\nArreglo después de ser ordenado:")
printArr(a)